app/assets/javascripts/paste_html_to_govspeak.js

```diff
@@ -0,0 +1,10 @@
+//= require paste-html-to-govspeak/dist/paste-html-to-markdown.js
+
+window.GOVUK = window.GOVUK || {};
+
+(function () {
+  function PasteHtmlToGovspeak (element) {
+    element.$el[0].addEventListener('paste', window.pasteHtmlToGovspeak.pasteListener)
+  }
+  GOVUK.PasteHtmlToGovspeak = PasteHtmlToGovspeak
+}(window.GOVUK))
```

Comment:
It looks like you've somewhat created your own module pattern here that is different to ones we use in GOV.UK. I think we can avoid it with the comment above.